doc/level1.rst

```diff
@@ -417,6 +417,12 @@ public keys.  It MAY support other OpenPGP key formats found in
 a ``type=1`` Autocrypt header (for example, by passing it agnostically
 to an OpenPGP backend for handling).
 
+We recommend against protecting the private key with a password. All
```

Comment:
i'd like to be specific: "MUAs SHOULD NOT protect the secret Autocrypt key with a password."